src/subsquare.ts

```diff
@@ -12,15 +12,15 @@ interface ReferendaObject {
   updatedAt: string;
   lastActivityAt: string;
   state: {
-    name: "Executed" | string;
+    name: "Executed" | "Rejected" | "TimedOut" | string;
   };
   onchainData: {
     /** Compare this to tx.method.hash.toHex() */
     proposalHash: string;
     timeline: {
       _id: string;
       referendumIndex: number;
-      name: "Executed" | "Confirmed" | "ConfirmStarted" | "DecisionStarted";
+      name: "Executed" | "Confirmed" | "ConfirmStarted" | "DecisionStarted" | "Rejected" | "TimedOut";
```

Comment:
What about `Killed` and `Cancelled`? :D

Reply:
I didn't find them so I hadn't add them. For now it would be simply ignoring them.

I remember that you showed me the code. I need to find it again to handle such cases.